mccurdy342/4fydp
# Reward_matrix

I expect a merge conflict initially and will resolve once a pull request is created. 

To test: 
1) Run 
2) Check output that each cohort 1-6 has a predominant home room teacher. 
3) Check that french is being satisified by french teachers 
4) Check teacher 10 is away
